modules/ui/init.js

```diff
@@ -249,6 +249,11 @@ export function uiInit(context) {
                 .call(uiAccount(context));
         }
 
+        aboutList
+            .append('li')
+            .attr('class', 'info-block')
+            .call(uiCoordinates(context));
+
```

Comment:
Looking at the bottom bar in the UI, I think the coordinate display would be more at home next to the scale bar on the left.  It also has the behavior of changing units when clicked.  I do like this better than the right-click menu, but please add a tool tip like in Hoot1, eg. "Click to change coordinate format".  Also I think MGRS should be one of the supported formats.